### PR TITLE
docs(avoidance): update documentation

### PR DESCRIPTION
**Pull request by satoshi-ota**
## Description

Update avoidance module documentation.

## Tests performed

Pass CI.

## Effects on system behavior

Nothing.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by go-sakayori** [comment]
I think there is an extra bracket ) after the phrase "path shifter"  in section Inner-workings / Algorithms

**Comment by go-sakayori** [comment]
In subsection "Judge if it's a parked vehicle", there is l_a in the equation and L_a in the description. Are they the same?

**Comment by go-sakayori** [comment]
Maybe the flowchart is not changed, but "Filtering flow for vehicle type objects" -> "Is stopping time longer than threshold" block has 2 no and 0 yes. Furthermore, there is a spelling miss in longher (it should be longer)

**Comment by satoshi-ota** [comment]
@go-sakayori Thank you for your pointing, and I fixed it. Please check it again :pray:

**Review by go-sakayori — COMMENTED** [review]
Sorry there were some comments pending .....

File: planning/autoware_behavior_path_static_obstacle_avoidance_module/README.md
[the remark this answers is not in the document]
Comment (satoshi-ota) [reply planning/autoware_behavior_path_static_obstacle_avoidance_module/README.md:1]: https://github.com/autowarefoundation/autoware.universe/pull/7149/commits/cd5955ea6c282cd4a6c57a9e62334fbefa67fa37

[the remark this answers is not in the document]
Comment (satoshi-ota) [reply planning/autoware_behavior_path_static_obstacle_avoidance_module/README.md:1]: https://github.com/autowarefoundation/autoware.universe/pull/7149/commits/cd5955ea6c282cd4a6c57a9e62334fbefa67fa37

[the remark this answers is not in the document]
Comment (satoshi-ota) [reply planning/autoware_behavior_path_static_obstacle_avoidance_module/README.md:1]: https://github.com/autowarefoundation/autoware.universe/pull/7149/commits/cd5955ea6c282cd4a6c57a9e62334fbefa67fa37

**Comment by satoshi-ota** [comment]
> Sorry there were some comments pending .....

Fixed suggested point in https://github.com/autowarefoundation/autoware.universe/pull/7149/commits/cd5955ea6c282cd4a6c57a9e62334fbefa67fa37.

**Review by go-sakayori — APPROVED** [review]

**Pull request merged**